_docs/_latest/configure/notifier/index.md

```diff
@@ -171,6 +171,36 @@ notifier:
             webhook_url: https://hooks.slack.com/services/foobar
 ```
 
+### CSCC Notification
+
+Forseti Security can configured to send violations to [Cloud Security Command Center (Cloud SCC)](https://cloud.google.com/security-command-center/).
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
```

Comment:
We removed the "/server/" folder in the GCS bucket and locally for inconvenience.

Reply:
Done.